examples/celery-rabbitmq/flower-controller.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  labels:
+    name: flower
+  name: flower-controller
+spec:
+  replicas: 1
+  selector:
+    name: flower
+  template:
+    metadata:
+      labels:
+        app: taskQueue
+        name: flower
```

Comment:
TBH I'm a bit confused as to how the labelling works here. When I tried to change this label to "component: flower", I got the error `ReplicationController "flower-controller" is invalid: spec.template.labels: invalid value 'map[app:taskQueue component:flower]': selector does not match template`. When I altered spec.selector.name to spec.selector.component, it ran without error, but it broke the example (the flower frontend is not reachable via the browser).

Is this a blocker?